pinot-broker/src/main/java/com/linkedin/pinot/broker/servlet/PinotBrokerTimeBoundaryDebugServlet.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.linkedin.pinot.broker.servlet;
+
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.linkedin.pinot.common.config.TableNameBuilder;
+import com.linkedin.pinot.common.utils.CommonConstants;
+import com.linkedin.pinot.routing.TimeBoundaryService;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/*
+ * A servlet to print the time boundary info for a table, if it exists.
+ */
+public class PinotBrokerTimeBoundaryDebugServlet extends HttpServlet {
+  private static final Logger LOGGER = LoggerFactory.getLogger(PinotBrokerTimeBoundaryDebugServlet.class);
+
+  private TimeBoundaryService _timeBoundaryService;
+
+  @Override
+  public void init(ServletConfig config) throws ServletException {
+    _timeBoundaryService = (TimeBoundaryService) config.getServletContext().getAttribute(TimeBoundaryService.class.toString());
+  }
+
+  @Override
+  protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+    String tableName = req.getPathInfo();
+    final String emptyResponse = "{}";
+    resp.setContentType("application/json");
+    boolean found = false;
+    try {
+      if (tableName.startsWith("/")) {
+        // Drop leading slash
+        tableName = tableName.substring(1);
+      }
+      if (!tableName.isEmpty()) {
+        CommonConstants.Helix.TableType tableType = TableNameBuilder.getTableTypeFromTableName(tableName);
+        if (tableType == null) {
```

Comment:
realtime does not have time boundary service (as of now). If we have something like that we can add code later.